dev/run-python-skinny-tests.sh

```diff
@@ -23,14 +23,14 @@ python -m pip install sqlalchemy alembic sqlparse
 # Given the example store does not delete dependencies, we verify non store related dependencies
 # after the example store setup. This verifies both the example store and skinny client do not add
 # unintended libraries.
-pytest --verbose tests/test_skinny_client_omits_data_science_libs.py
-
-pytest --verbose tests/test_runs.py
-pytest --verbose tests/tracking/test_client.py
-pytest --verbose tests/tracking/test_tracking.py
-pytest --verbose tests/projects/test_projects.py
-pytest --verbose tests/deployments/test_cli.py
-pytest --verbose tests/deployments/test_deployments.py
-pytest --verbose tests/projects/test_projects_cli.py
+pytest tests/test_skinny_client_omits_data_science_libs.py
+
+pytest tests/test_runs.py
+pytest tests/tracking/test_client.py
+pytest tests/tracking/test_tracking.py
+pytest tests/projects/test_projects.py
+pytest tests/deployments/test_cli.py
+pytest tests/deployments/test_deployments.py
+pytest tests/projects/test_projects_cli.py
```

Comment:
then why not add --verbose here ?

Reply:
Because we no longer have to, please see this comment:
https://github.com/mlflow/mlflow/pull/3759/files#r547026831

Reply:
As a proof, you can check this `python-skinny` test logs:

https://github.com/mlflow/mlflow/pull/3759/checks?check_run_id=2683329790#step:4:1